miio/device.py

```diff
@@ -115,7 +115,7 @@ class Device:
     This class should not be initialized directly but a device-specific class inheriting
     it should be used instead of it."""
     def __init__(self, ip: str = None, token: str = None,
-                 start_id: int=0, debug: int=0, lazy_discover: bool=True) -> None:
+                 start_id: int=0, debug: int=0, lazy_discover: bool=True, model: str=None) -> None:
```

Comment:
Is it truly necessary to introduce this also on the parent level, or would it be fine to have it just in the subclass?